Makefile.am

```diff
@@ -55,29 +58,19 @@ install-data-hook:
 	mv $(DESTDIR)$(datadir)/@PACKAGE@/test $(DESTDIR)$(datadir)/@PACKAGE@/tests; \
 	cp test/testcases/xmlonly.sh $(DESTDIR)$(datadir)/@PACKAGE@/tests/testcases/configbasic-xml.filter
 
-# Python module installation
+# Python module installation using modern tools
 all-local:
-	(cd $(srcdir); $(PYTHON) setup.py build \
-		--build-base $(shell readlink -f $(builddir))/build \
-		--verbose)
-
-python_prefix = --prefix=$(prefix)
+	(cd $(srcdir); python3 -m build --outdir $(shell readlink -f $(builddir))/dist --wheel --sdist --verbose)
```

Comment:
As you write change this into a single line, you will not need to use a sub shell any longer.